civictechprojects/models.py

```diff
@@ -431,6 +443,10 @@ def hydrate_project_volunteer_info(self):
 
     def update_project_timestamp(self):
         self.project.save()
+
+    def is_up_for_renewal(self, now=None):
+        now = now or timezone.now()
+        return (self.projected_end_date - now) < settings.VOLUNTEER_REMINDER_OVERALL_PERIOD
```

Comment:
In my dev environment I get an error pointing to this line every time I attempt to log in. I've run the migrations so I'm not sure what it is:

```
TypeError at /index/

unorderable types: datetime.timedelta() < NoneType()

Request Method: 	GET
Request URL: 	http://127.0.0.1:8000/index/?section=FindProjects&showSplash=1
Django Version: 	1.10.7
Exception Type: 	TypeError
Exception Value: 	

unorderable types: datetime.timedelta() < NoneType()

Exception Location: 	/home/dev-admin/git/CivicTechExchange/civictechprojects/models.py in is_up_for_renewal, line 449
```

Reply:
Figured it out, my fault. Forgot to set the new env variables.